modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -4,41 +4,53 @@
-+
-[NOTE]
-====
-Ensure that you do not select the *Community* version of the *OpenShift Pipelines Operator*.
-====
-+
-//image::op-installed-tile.png[]
-//Remove the note below once the community operator is removed from OperatorHub
+. Use the *Filter by keyword* box to search for `{pipelines-title}` Operator in the catalog. Click the *{pipelines-title}* Operator tile.
 
-. Read the brief description about the Operator on the *{pipelines-title} Operator* page. Click *Install*.
-+
-//image::op-install-subscription.png[]
+. Read the brief description about the Operator on the *{pipelines-title}* Operator page. Click *Install*.
 
 . On the *Install Operator* page:
++
+.. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+
+.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+
+.. Select an *Update Channel*.
 
-  .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+*** The *ocp-<4.x>* channel enables installation of the latest stable release of the {pipelines-title} Operator.
+*** The *preview* channel enables installation of the latest preview version of the {pipelines-title} Operator, which may contain features that are not yet available from the 4.x update channel.
+
++
+[NOTE]
+====
+The {pipelines-title} Operator now provides the option to choose the components that you want to install by specifying profiles.
+The supported profiles are:
 
-  .. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
-  .. Select an *Update Channel*.
-    *** The *ocp-<4.x>* channel enables installation of the latest stable release of the {pipelines-title} Operator.
-    *** The *preview* channel enables installation of the latest preview version of the {pipelines-title} Operator, which may contain features that are not yet available from the 4.x update channel.
+* Basic: This installs only Tekton pipelines.
+* Default: This installs Tekton pipeline and Tekton triggers.
+* All: This installs all of the Tekton components.
```

Comment:
@Preeticp as `TektonConfig` CR will be auto installed when Operator is installed 
So we may need to change/add a word here that by default the used profile is `All`

Reply:
Do we mention anywhere what are the Tekton components available?

Reply:
ya that would make sense
when 
`All: This installs all of the Tekton components.`
@Preeticp here `Tekton components` means
Tekton Pipelines, Tekton Triggers, Tekton Addons(which include ClusterTasks, ClusterTriggerBindings, ConsoleCLIDownload, ConsoleQuickStart and ConsoleYAMLSample)